frontend/src/Startups/startup.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import s1 from '../Images/start/1.png';
+import s2 from '../Images/start/2.jpg';
+import s3 from '../Images/start/3.jpeg';
+import s4 from '../Images/start/4.jpg';
+import '../Startups/startup.css';
+
+const startup=()=> (

```

Comment:
Better make a react component what say @naveennvrgup ?

Reply:
@kdpisda I had a video call with frontend guys to not do this immediately after this commit. explained them how to fetch this from our apis and do this dyanamically the react way.